### DISCUSSION TITLE
how to use harfbuzz installed by julia lang's pkg used system wide?

### DISCUSSION THREAD

**Discussion by goyalyashpal** [harfbuzz/harfbuzz#3786] [General] [answered]
So, on Julia REPL, i did:

```julia
import Pkg; Pkg.add( "Plots" )
```

and i spotted these along the lines:

`HarfBuzz_jll v2.8.1+1`

so, my question is, 
* how can that be used in my main system? 

+ like i would need harfbuzz for sile, 
+ harfbuzz seems to be not distributed in binary for windows
+ i am not familiar with meson builds
+ so... it would be really awesome if i could use a harfbuzz which already exists on my system rather than trying to relearning the wheel

**Comment by behdad** [comment]
> harfbuzz seems to be not distributed in binary for windows

We actually do distribute win32 binaries.

**Comment by ebraminio** [comment, marked as answer]
Guess you can direct your questions to https://github.com/JuliaBinaryWrappers/HarfBuzz_jll.jl the organization doesn't seem to be defunct yet https://github.com/JuliaBinaryWrappers

> i am not familiar with meson builds

You can just use src/harfbuzz.cc, no build system required even.
